WMF Framework/WMFOnThisDayEventsFetcher.m

```diff
@@ -27,9 +27,13 @@ - (BOOL)isFetching {
     return [[self.operationManager operationQueue] operationCount] > 0;
 }
 
+- (NSArray*)supportedLanguages {
```

Comment:
This should return an `NSSet` instead of an array due it's use as a lookup. It should also be a class method with a `static NSSet` created within a `dispatch_once` block.